ros/api/v0/hosts.py

```diff
@@ -268,7 +272,7 @@ class HostHistoryApi(Resource):
     performance_utilization_fields = {
         'cpu': fields.Integer,
         'memory': fields.Integer,
-        'io': fields.Integer,
+        'io': fields.Raw,
```

Comment:
```suggestion
        'io_all': fields.Raw,
```
To be consistent, let's keep `io_all`. what do you think?

Reply:
Yes true, but I didn't want to introduce too many new changes on the front-end, in case they were not required.
Shall I add the max_io and io_all to HostHistoryApi as well?
@PreetiW thoughts on :arrow_up: ?

Reply:
@saltgen - currently, ros-frontend is not consuming HostHistory API.

Reply:
+1 for `io_all`

Reply:
@saltgen @kgaikwad is correct, I checked ros-frontend and we are not consuming HostHistoryApi anywhere. I would suggest here to be consistent and add both `max_io` and `io_all` to HostHistoryApi in case we will consume it in future.

Reply:
ah cool, thanks @kgaikwad @PreetiW 
 
Response from HostHistory should have `max_io` and `io_all` now. Changes pushed :bow_and_arrow: